runelite-mixins/src/main/java/net/runelite/mixins/GameScalingMixin.java

```diff
@@ -96,13 +105,58 @@ public void setStretchedKeepAspectRatio(boolean state)
 		cachedStretchedDimensions = null;
 	}
 
+	@Inject
+	@Override
+	public void setScalingFactor(int factor)
+	{
+		if (factor > 100)
```

Comment:
```
factor = Math.min(Math.max(factor, 0), 100)
```

Reply:
Changed to use ``Ints.constrainToRange(factor, 0, 100);``